### PR TITLE
Redesign Post[others] and Post[self]- New UI

### PR DESCRIPTION
**Pull request by SamriddhiMishra**
### Screenshots of the changes (If any) -
1)

Screenshot (647)

2)
Screenshot (649)

The comment section is not resolved.

### CODE REVIEW

**Comment by nitin10s** [comment]
@SamriddhiMishra Can you please post the screenshots of the mobile view as well? The web view looks great!

**Comment by SamriddhiMishra** [comment]
@nitin10s 
Screenshot (651)
Screenshot (650)

**Comment by SamriddhiMishra** [comment]
@nitin10s Is this alright?
Screenshot (654)
Screenshot (655)

**Comment by SamriddhiMishra** [comment]
@nitin10s Is this alright?
Screenshot (657)
Screenshot (656)

**Comment by nitin10s** [comment]
@SamriddhiMishra Can you please fix the conflicts?

**Comment by tachyons** [comment]
Fix merge conflicts, also ensure 2 space indentation in erb files

File: app/assets/stylesheets/theme.scss [hunk 1]
```diff
@@ -17,5 +18,9 @@ $input-btn-focus-width: .2rem;
 $color-hover: $green;
 $active-background: white;
 $active-border: $teal;
+$btnselfborder: #2f8875;
+$acolor: #0082ff;
```
Comment (tachyons) [inline app/assets/stylesheets/theme.scss]: link-color

File: app/assets/stylesheets/theme.scss [hunk 2]
```diff
@@ -17,5 +18,9 @@ $input-btn-focus-width: .2rem;
 $color-hover: $green;
 $active-background: white;
 $active-border: $teal;
+$btnselfborder: #2f8875;
```
Comment (tachyons) [inline app/assets/stylesheets/theme.scss]: `button-border-color` ?

Comment (SamriddhiMishra) [reply app/assets/stylesheets/theme.scss]: Yes, when seeing projects of self.

Comment (aayush-05) [reply app/assets/stylesheets/theme.scss]: I think, he means that maybe you can use a little better naming convention.

**Comment by tachyons** [comment]
Screenshot 2020-03-02 at 10 37 03 AM
I am still seeing lot of issues which are not addressed

**Comment by tachyons** [comment]
Install codeclimate browser extension

**Comment by nitin10s** [comment]
@SamriddhiMishra Are you working on fixing this? Please acknowledge.

**Comment by SamriddhiMishra** [comment]
@tachyons Can you please give me a clue why this AccessLint check is pending or how to solve this?

**Pull request merged**